tools/scylla-nodetool.cc

```diff
@@ -365,6 +370,137 @@ void compactionhistory_operation(scylla_rest_client& client, const bpo::variable
     }
 }
 
+class Tabulate {
```

Comment:
Maybe I should make `compactionhistory` use this too (it formats a table too).

Reply:
i can refactor it later on.

Reply:
i just took a look at `compactionhistory`. despite that it always uses `{:<}` for headers like C* nodetool does, it customizes the alignment on a per-column basis. we could use different alignment for different types, like `{:<}` for string alike fields, and `{:>}` for numeric fields, or allow the caller to customize the format string. either way, the current `Tabulate` implementation needs more work to adapte it to `compactionhistory`.